taipy/core/data/parquet.py

```diff
@@ -51,7 +51,7 @@ class ParquetDataNode(DataNode, _FileDataNodeMixin, _TabularDataNodeMixin):
         parameters which are also passed to Pandas.
     - *"write_kwargs"* (`Optional[dict]`): Additional parameters passed to the
```

Comment:
The rule is to reference parameters in italics only, no double quotes.
Something to strengthen in the future, hu?

Reply:
fixed